src/content/docs/new-relic-control/agent-control/migration.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: "Agent Control migration guide"
+metaDescription: "You can reinstall your current instrumentation to manage it through Agent Control"
+freshnessValidatedDate: never
+---
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+## Migrate the instrumentation to manage it with Agent Control
+If you have a cluster already instrumented with NewRelic you can decide to remove the existing agents and manage them through the Agent Control.
+All features and all configs exposed by the HelmChart of each Agent can still be applied in a unified way through the Agent Control.
+
+### Match the existing instrumentation with the new Agents managed by the AgentControl
+First of all you need to match your existing instrumentation with the supported agents.
+
+In particular you can deploy and manage through the Agent Control:
+  - Most of the Helm Charts included in the `nri-bundle` chart: 
+    - `newrelic-infrastructure`, `nri-kube-events`, `kube-state-metrics`, and `nri-kube-events` now managed through the New Relic Infrastructure AgentType.
```

Comment:
Maybe adding here a link to what is an agent_type, do we have that?

Reply:
I think that should be addressed in the main PR